package.json

```diff
@@ -29,6 +29,7 @@
     "access": "public"
   },
   "dependencies": {
+    "@carbon/react": ">1.47.0 <1.50.0",
```

Comment:
@samuelmale Is there a specific reason you made this a dependency and not a peerDependency?

I've read the linked issues/pr and the issue with the downshift dependency in carbon which has since been resolved. After pulling the latest changes and linking with the form-builder, i got some errors from carbon because of version conflicts. 


1. On one end adding this range resolved the issue with the downshift library but it also creates either a conflict or multiple versions of `@carbon/react` running within microfrontends that have dependencies out of that range for example esm-form-builder and other esms are currently running on `@carbon/react@npm:1.37.0` which will in turn affect the bundle size. Would this necessitate us updating the frontends that pull the FE directly to at least `1.47.0` to have this fix working?

<img width="666" alt="Screenshot 2024-06-21 at 11 02 10" src="https://github.com/openmrs/openmrs-form-engine-lib/assets/15266028/68d5c775-ce06-42de-9043-104d38ab7866">


2. Doesn't having the same range of versions as a peer dependency yield the same result but with the benefit of not duplicating instances of `@carbon/react`?

Reply:
IIRC, the overwhelming consensus on the dependency management strategy for Carbon in O3 is having it as a normal dependency as opposed to a singleton (peer).